app/views/facility_journals/index.html.haml

```diff
@@ -29,12 +29,7 @@
             %tr
               %td
                 %ul.unstyled
-                  - if Settings.financial.journal_format.xls
-                    %li=link_to 'XLS File', pending_journal.file.url
-                  - if Settings.financial.journal_format.xml
-                    %li=link_to 'XML File', facility_journal_path(current_facility, pending_journal, format: 'xml')
-                  - if Settings.financial.journal_format.csv
-                    %li=link_to 'CSV File', facility_journal_path(current_facility, pending_journal, format: 'csv')
+                  = render "downloads", facility: current_facility, journal: pending_journal
```

Comment:
Using the same locales here results in too much text in the column. I think we want to leave this cell as just "XML", "XLS", "CSV". The other view should have the full "Download XXX file".